src/main/webapp/i18n/en/exam.json

```diff
@@ -561,6 +561,7 @@
                 "titleInfo": "The title of the exercise group is shown to the students as the name of the exercise.",
                 "delete": {
                     "question": "Are you sure you want to delete the exercise group <strong>{{ title }}</strong>? All associated exercises will also be deleted.",
+                    "questionLocalVC": "Are you sure you want to delete the exercise group <strong>{{ title }}</strong>? All associated exercises will be deleted.<br>All student, template, solution, test, and auxiliary repositories and build plans will also be deleted.<br>Be careful: this can <strong class='text-danger'>NOT</strong> be undone!<br>Before you activate this option, we recommend that you archive the exercise first! ",
```

Comment:
I think this should be "cannot" as a single word instead of "can not". I vaguely recall that this has gotten us in trouble during an EIST quiz before 😅

Reply:
what was the problem in the EIST quiz? 😄 If there are some nuances with 'cannot' and 'can not' that I'm now aware of, I will replace all occurrences of 'can not', but for me personally they mean the same. Multiple sources also confirm that.

Reply:
Basically students argued that it could mean "can (maybe) not". This isn't exactly accurate imo, but it was enough to invalidate the question back then.

The real nuance between the spellings "cannot" and "can not" lies in its usage: we should use "cannot" when the "not" is not part of an adjacent phrase like "not only".

You'd write

"You cannot do this."

and

"You can not only do this, but also that."

Reply:
If you'd really rather not change it, then don't - I'll leave it up to you. It's just what I've learned about the use of cannot vs. can not.